public/app/percona/pmm-dump/PMMDump.service.ts

```diff
@@ -38,6 +38,24 @@ export const PMMDumpService = {
   async delete(dumpIds: string[]) {
     await api.post<void, DeleteDump>(`${BASE_URL}/Delete`, { dump_ids: dumpIds });
   },
+  async dowload(dumpIds: string[]) {
+    for (const dumpId of dumpIds) {
+      api.get<AxiosResponse<Blob>, DeleteDump>(`/dump/${dumpId}.tar.gz`).then((response: AxiosResponse<Blob>) => {
```

Comment:
Any reason to use `.then` and not async/await?

If the get request fails for whatever reason will the user be notified? I think there may be a possibility of the get request failing and withAppEvents not triggering.